ci/pipelines/ova-release.yml

```diff
@@ -98,6 +104,14 @@ jobs:
       params:
         file: dispatch-logs/*.tar.gz
 
+- name: build-dispath-ui
+  public: true
+  plan:
+  - get: dispatch-ui
+    resource: dispatch-ui
+  - task: build-ui
+    file: dispatch-ui/ci/tasks/build-bundle.yml
```

Comment:
Is this file missing from the commit?

Reply:
No, it located in ui repo
https://github.com/dispatchframework/dispatch-ui/blob/master/ci/tasks/build-bundle.yml